components/tutorial/tutorial_tip.jsx

```diff
@@ -16,31 +16,28 @@ import tutorialGif from 'images/tutorialTip.gif';
 import tutorialGifWhite from 'images/tutorialTipWhite.gif';
 
 const Preferences = Constants.Preferences;
+const TutorialSteps = Constants.TutorialSteps;
 
 export default class TutorialTip extends React.Component {
     constructor(props) {
         super(props);
 
         this.handleNext = this.handleNext.bind(this);
-        this.toggle = this.toggle.bind(this);
+        this.show = this.show.bind(this);
```

Comment:
We prefer to use arrow functions as class methods and remove the bind lines in the constructor

Reply:
Makes sense -- done!